prow/config.yaml

```diff
@@ -5979,6 +5979,21 @@ presubmits:
         name: auto-generated-docker-graph-volume-mount
     trigger: (?m)^/test pull-security-kubernetes-verify-prow,?(\s+|$)
   kubernetes-sigs/cluster-api:
+  - name: pull-cluster-api-all
```

Comment:
For the same reasons as config-json, please move this below pull-cluster-api-build.

Reply:
same as above, i believe we will fail some sanity checks if the file is not sorted

Reply:
Yep. For now auto sort is enforced and there is a script to fixup the file to sorted. I plan to eliminate this once the jobs are split into multiple files (probably per repo/sig)